src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -696,16 +696,18 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
 
     std::map<std::string, ASR::ttype_t*> implicit_dictionary;
     std::map<uint64_t, std::map<std::string, ASR::ttype_t*>> &implicit_mapping;
+    bool &is_Function;
```

Comment:
```suggestion
    bool &is_function;
```
And move it above with the other such bool variables.